qoajad_backend/src/main/java/com/qoajad/backend/controller/DefaultController.java

```diff
@@ -0,0 +1,93 @@
+package com.qoajad.backend.controller;
+
+import com.qoajad.backend.model.User;
+import com.qoajad.backend.service.user.UserService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Qualifier;
+import org.springframework.cloud.context.config.annotation.RefreshScope;
+import org.springframework.dao.DataIntegrityViolationException;
+import org.springframework.dao.EmptyResultDataAccessException;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+import java.util.List;
+
+@RefreshScope
+@RestController
+public class DefaultController {
```

Comment:
Another good point of start is to change this class name for something along the lines of UserController. Following SOLID
S (Single Responsibility): This class should be responsible for endpoints that have to do with an User